pr_check.sh

```diff
@@ -13,6 +13,18 @@ done <<< "$(git log --pretty=format:%s $(git merge-base master HEAD)..HEAD)"
 
 set -exv
 
+python3 -m venv .venv
+source .venv/bin/activate
+pip3 install sphinx-rtd-theme
+
+cd docs/
+make clean
+make html
+
+deactivate
+
+cd -
```

Comment:
These are great as one use, but I've seen em cause some mighty headaches in the past. Just a heads up if this is modified in the future or moved to a function.

Reply:
Maybe just keep it all on one line. `cd docs && make clean && make html && cd -`

then call `deactivate` on the next line?

Reply:
I like that